app/services/hyrax/queries.rb

```diff
@@ -5,15 +5,15 @@ class Queries
     class_attribute :metadata_adapter
     self.metadata_adapter = Valkyrie.config.metadata_adapter
     class << self
-      delegate :find_all, :find_by, :find_members, :find_inverse_references_by, to: :default_adapter
+      delegate :find_all, :find_all_of_model, :find_by, :find_members, :find_inverse_references_by, to: :default_adapter
```

Comment:
If `Hyrax::Queries` is going to be a standard practice, we should have its specs run the shared specs.

Reply:
I can work on that.  Do you think it should go in this PR?